packages/components/AssetsPagination.tsx

```diff
@@ -0,0 +1,177 @@
+import { SpacerRow } from "./spacer";
+import chevronLeftDoubleSVG from "../../assets/icons/chevron-left-double.svg";
+import chevronLeftSVG from "../../assets/icons/chevron-left.svg";
+import chevronRightDoubleSVG from "../../assets/icons/chevron-right-double.svg";
+import chevronRightSVG from "../../assets/icons/chevron-right.svg";
+import {
+  neutral17,
+  neutral33,
+  neutral77,
+  primaryColor,
+} from "../utils/style/colors";
+import { fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
+
+interface PaginationProps {
+  currentPage: number;
+  maxPage: number;
+  itemsPerPage: number;
+  dropdownOptions: number[];
+  setItemsPerPage: (item: number) => void;
+  onChangePage: (page: number) => void;
+}
+
+export const AssetsPagination = ({
+  currentPage,
+  maxPage,
+  dropdownOptions,
+  itemsPerPage,
+  setItemsPerPage,
+  onChangePage,
+}: PaginationProps) => {
+  const handleChangePage = (pageIndex: number) => {
+    if (pageIndex < 0) {
+      pageIndex = 0;
+    } else if (pageIndex >= maxPage) {
+      pageIndex = maxPage - 1;
+    }
+    if (pageIndex !== currentPage) {
+      onChangePage(pageIndex);
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={{ flex: 1, flexDirection: "row" }}>
+        <View style={[styles.section, { justifyContent: "flex-start" }]}>
+          <BrandText style={styles.grayText}>
+            Page {currentPage + 1} of {maxPage}
+          </BrandText>
+        </View>
+        <View style={[styles.section, { justifyContent: "flex-start" }]}>
+          <BrandText style={styles.grayText}>| Go to page:</BrandText>
+          <TertiaryBox
+            style={[
+              {
+                flexDirection: "row",
+                paddingHorizontal: 12,
+                backgroundColor: neutral17,
+                width: 80,
+                height: SEARCH_BAR_INPUT_HEIGHT,
+                alignItems: "center",
+              },
+            ]}
+          >
+            <TextInput
+              value={currentPage.toString()}
+              inputMode="numeric"
+              style={[
+                fontSemibold14,
+                {
+                  color: "white",
+                  flex: 1,
+                  width: 56,
+                },
+                { outlineStyle: "none" } as any,
+              ]}
+              onChangeText={(text) => {
+                handleChangePage(Number(text));
+              }}
+            />
+          </TertiaryBox>
+        </View>
+      </View>
+      <SpacerRow size={1} />
+      <View style={{ flex: 1 }}>
+        <View style={styles.section}>
+          <TouchableOpacity onPress={() => handleChangePage(0)}>
+            <LegacyTertiaryBox height={42} width={56}>
+              <SVG source={chevronLeftDoubleSVG} height={16} width={16} />
+            </LegacyTertiaryBox>
+          </TouchableOpacity>
+          <SpacerRow size={1} />
```

Comment:
Maybe we should add blank line between block code to make the code more readable :)

Ex: 

```
<View>...</View>

<SpacerRow size={1} />

<View>...</View>
```

This is just a suggestion

Reply:
Okay. Thanks for the suggestion.